src/components/pages/Volunteers/AddContactModal.tsx

```diff
@@ -0,0 +1,410 @@
+import React, { useState } from 'react';
+import { Button, Form, Modal, Col } from 'react-bootstrap';
+import { submitButton } from 'src/redux/modules/volunteerContacts';
+import { connect, ConnectedProps } from 'react-redux';
+import { bindActionCreators, Dispatch } from 'redux';
+import { RootState } from '../../../redux';
+
+interface AddContactModalProps {
+  volunteer: Volunteer;
+  show: boolean;
+  handleClose: () => void;
+  token: string;
+  orgId: number;
+}
+const mapStateToProps = (
+  state: RootState,
+  addContactModalProps: AddContactModalProps,
+) => ({
+  vcState: state.volunteerContacts,
```

Comment:
so this is also a technical debt that we accumulated, but ideally, our components should never have to access the store.

Ideally this would be passed down from the respective container (index.ts) and all components are dummy.

Sarah took a stab but she was struggling to pass down the function from the container. 

This is an example of how I did it on Connect: https://github.com/Ameelio/connect-videoconferencing-client/pull/1/files#diff-c0502a25855c2a8f255cec809139f3f2R42

Let me know what u think!